cmd/ocm-backplane/login/login.go

```diff
@@ -72,33 +85,125 @@ func init() {
 		"Save kube configuration in the specific path when login to multi clusters.",
 	)
 
+	flags.StringVar(
+		&args.pd,
```

Comment:
I feel use `alert` instead of `pd` in the parameter should be better

Reply:
Word 'alert' could be misleading. Current logic takes incident as input. Option for alert as input will be added in future.

With that being said I would suggest to keep generic arg name.